src/challenges/ch-1/ch-1.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Function to calculate simple interest.
+ * @param principle number
+ * @param time number
+ * @param rate number
+ *
+ * @return SI (Simple Interest) number
+ */
+export function calculateSimpleInterest(
+  principle: number,
+  time: number,
+  rate: number
+) {
+  const SI = (principle * time * rate) / 100;
```

Comment:
Let's use lowercase to represent the result value.
Would be better to use `simpleIntereset` instead of the `SI` as uppercase is used to represent the final constant in most of the cases.